easybuild/easyconfigs/b/Boost/Boost-1.70.0-foss-2019a.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Boost'
+version = '1.70.0'
+
+homepage = 'http://www.boost.org/'
+description = "Boost provides free peer-reviewed portable C++ source libraries."
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
why not use the existing easyconfig for `gompi`, which is a subtoolchain of `foss`?

Reply:
You're right, that's absolutely not necessary. I didn't know that `gompi` was a subtoolchain of `foss`.